### DISCUSSION TITLE
Softkeyboard dimensions

### DISCUSSION THREAD

**Discussion by devbinjoseph** [airsdk/Adobe-Runtime-Support#3461] [General]
I would like to place UI elements aligned with the top of whatever keyboard is shown (including if it changes, such as to emojis). Is there any facility to read the softkeyboard dimensions and/or placement in iOS? Android?

**Comment by Devin-Licastro** [comment]
I personally use this ANE from Distriqt. 
https://airnativeextensions.com/extension/com.distriqt.Application/

It lets you do something like: keyboardHeight = Application.service.keyboard.height / Starling.current.contentScaleFactor;
The keyboard needs to be opened once before you can get its height though, so the first time the UI elements are displayed you should delay displaying them until you're sure the keyboard is all the way open, then run the height check and position them. 

Works on both Android and iOS

**Reply by devbinjoseph** [reply]
Thanks @Devin-Licastro , I'll have a look at this.
